### PR TITLE
chore: Improve l10n build performance

### PR DESCRIPTION
**Pull request by GeoffCoxMSFT**
## Description

The l10n:babel extracts strings from the source code. 
It is currently including lib folders which contain built JS with the same formatMessage calls as the TS source.
This creates a lot of extra files in the l10ntemp folder which later l10n commands have to process - slowing the build.

This PR excludes those folders using the wildcard search pattern in the ignore section of the babel config file.
I tried to use the regex format, but it doesn't seem to work and babel won't output information about matches.

## Task Item

#minor 

## Screenshots

Old yarn l10n build time (on my dev machine): 115 s
New yarn l10n build time (on my dev machine): 95 s
Savings: 20 s

### CODE REVIEW

File: Composer/babel.l10n.config.js [hunk 1]
```diff
@@ -1,13 +1,34 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// TODO: It would be nice to use the regular expression matching.
+// It would be nice to ignore the lib folders rather than listing them one at a time.
+// However, as of 4/2021 it doesn't seem to work.
+// We have the added complexity that packages/lib contains folders we DO want to process.
```
Comment (a-b-r-o-w-n) [inline Composer/babel.l10n.config.js:7]: Yeah. Easiest thing to do is probably just remove the lib directory and move the contents up a directory.

**Review by a-b-r-o-w-n — APPROVED** [review]
@beyackle can you also take a look at this?

File: Composer/babel.l10n.config.js (hunk 1 above)
Comment (GeoffCoxMSFT) [reply Composer/babel.l10n.config.js:7]: Agree.  Didn't want to create any major restructure so close to release.

Comment (beyackle) [reply Composer/babel.l10n.config.js:7]: I definitely agree here, too. It keeps tripping me up that we have regular non-compiled code in a `lib` folder (and then each package in there has compiled stuff in its own `lib` folder).

**Review by beyackle — APPROVED** [review]

**Pull request merged**